src/gui/network.go

```diff
@@ -8,6 +8,64 @@ import (
 	wh "github.com/skycoin/skycoin/src/util/http" //http,json helpers
 )
 
+// ConnectionStatus structs
+type ConnectionStatus struct {
+	Status     string `json:"Status"`
+	Connection string `json:"Connection"`
+	isAlive    bool   `json:"isalive"`
```

Comment:
Name should start with capital `I` i.e. `IsAlive` for this field to be included in response .